src/test/groovy/org/ajoberstar/gradle/git/release/semver/NearestVersionLocatorSpec.groovy

```diff
@@ -15,18 +15,20 @@
  */
 package org.ajoberstar.gradle.git.release.semver
 
-import java.nio.file.Files
-import java.security.SecureRandom
-
 import com.github.zafarkhaja.semver.Version
-
 import org.ajoberstar.grgit.Grgit
-
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 import spock.lang.Shared
 import spock.lang.Specification
 import spock.lang.Unroll
 
+import java.nio.file.Files
+import java.security.SecureRandom
+
 class NearestVersionLocatorSpec extends Specification {
```

Comment:
Just so the benchmark could get at them and reuse them, without requiring an instance of the spec.